server/app/services/settings/SettingsManifest.java

```diff
@@ -955,6 +955,11 @@ public boolean getFastforwardEnabled(RequestHeader request) {
     return getBool("FASTFORWARD_ENABLED", request);
   }
 
+  /** Enables civiform admins to set up a customized eligibility message per screen. */
+  public boolean getEligibilityMessageEnabled(RequestHeader request) {
+    return getBool("ELIGIBILITY_MESSAGE_ENABLED", request);
```

Comment:
**nitpick:** It may be clearer to call this `CUSTOMIZED_ELIGIBILITY_MESSAGE_ENABLED`

Reply:
sure thing. done!